src/sql/sqlcmd.py

```diff
@@ -58,6 +59,8 @@ def sqlcmd_snippets(others):
         help="Force delete all stored snippets",
         required=False,
     )
+    if len(others) == 1 and others[0] in util.get_all_keys():
```

Comment:
let's show a more informative error. currently:

```
In [7]: %sqlcmd snippets not-a-snippet
UsageError: unrecognized arguments: not-a-snippet
```

it'd be better to show:

```
UsageError: 'not-a-snippet' is not a snippet. Available snippets are A, B, and C
```

Reply:
> let's show a more informative error. currently:
> 
> ```
> In [7]: %sqlcmd snippets not-a-snippet
> UsageError: unrecognized arguments: not-a-snippet
> ```
> 
> it'd be better to show:
> 
> ```
> UsageError: 'not-a-snippet' is not a snippet. Available snippets are A, B, and C
> ```

Okay. I will work on that after resolving the commit error for this PR.

Reply:
I think your PR is fine, the error I see in the logs is:

```
 FAILED src/tests/test_ggplot.py::test_ggplot_geom_histogram_with[png] - _tkinter.TclError: Can't find a usable init.tcl in the following directories: 
    {C:\hostedtoolcache\windows\Python\3.11.4\x64\tcl\tcl8.6}

C:/hostedtoolcache/windows/Python/3.11.4/x64/tcl/tcl8.6/init.tcl: couldn't read file "C:/hostedtoolcache/windows/Python/3.11.4/x64/tcl/tcl8.6/init.tcl": No error
couldn't read file "C:/hostedtoolcache/windows/Python/3.11.4/x64/tcl/tcl8.6/init.tcl": No error
    while executing
"source C:/hostedtoolcache/windows/Python/3.11.4/x64/tcl/tcl8.6/init.tcl"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 [list source $tclfile]"

```

this is an intermittent error, if you push an empty commit to trigger the CI, it should pass:

```
git commit --allow-empty -m ci
```

Reply:
@bbeat2782 
I re-ran it and it passes